waku/v2/protocol/waku_rln_relay/waku_rln_relay_metrics.nim

```diff
@@ -1,9 +1,20 @@
+{.push raises: [Defect].}
+
 import
+  chronicles,
+  chronos,
   metrics,
-  waku_rln_relay_constants
+  metrics/chronos_httpserver,
+  waku_rln_relay_constants,
+  ../../utils/collector
 
 export metrics
 
+const LogPeriod = 15
```

Comment:
Please indicate the unit.

Reply:
Why not use the same timer duration as the other waku metrics logging i.e., `30`s?

Reply:
Just providing flexibility if it needs to be changed later.

Reply:
See my comment above on flexibility. I don't think we should be eager to add "in case" flexibility if it's unclear that such a need will ever arise.